apps/finance/test/finance.js

```diff
@@ -1,3 +1,4 @@
+const assertEvent = require('@aragon/test-helpers/assertEvent')
```

Comment:
I think we should check revert reasons in this contract to ensure that things are reverting with the error that we expected. See https://github.com/aragonone/aragon-court/blob/6d0b322becfb5ff528437c14087ebd07cc918520/test/helpers/assert-revert.js which uses Truffle 4